lib/Transforms/Instrumentation/ExtendedPointer.cc

```diff
@@ -133,44 +150,99 @@ void ExtendedPointer::emitBranchToTrap(Value *Cmp) {
 bool ExtendedPointer::instrument(Value *Ptr, Value *InstVal,
                                 const DataLayout &DL) {
   uint64_t NeededSize = DL.getTypeStoreSize(InstVal->getType());
+  Type *IntTy = DL.getIntPtrType(Ptr->getType());
+  Value *NeededSizeVal = ConstantInt::get(IntTy, NeededSize);
   DEBUG(dbgs() << "Instrument " << *Ptr << " for " << Twine(NeededSize)
                << " bytes\n");
 
   SizeOffsetEvalType SizeOffset = ObjSizeEval->compute(Ptr);
-
+  Value *Or;
   if (!ObjSizeEval->bothKnown(SizeOffset)) {
-    ++ChecksUnable;
-    return false;
+    WARN << "fatptr instrument " << *Ptr << " for " << Twine(NeededSize) << " bytes\n";
```

Comment:
protect this print by `LOG`. 

This does not look like a warning message to me, more like a regular log

Reply:
fixed